backend/src/main/java/aiversum/backend/rest/ChatController.java

```diff
@@ -77,6 +91,16 @@ public Flux<String> generateOllama(String model, String messages) {
 
         return generateResponse(messages, streamingChatLanguageModel);
     }
+    public Flux<String> generateVertexai(String model, String messages){
+        StreamingChatLanguageModel streamingChatLanguageModel = VertexAiGeminiStreamingChatModel.builder()
+                .project("") /** The PROJECT field represents the variable you set when creating a new Google Cloud project. */
```

Comment:
This needs to be moved to application.properties as well, because it won't work without this variable